docs/dev-standards/cloud-security-alignment.md

```diff
@@ -0,0 +1,35 @@
+# Cloud security alignment
+
+## Introduction
+
+Cloud services **MUST** be designed and operated securely to protect sensitive data, maintain service availability and support public trust.
+
+This guidance helps teams build secure cloud-native systems by aligning with cloud provider best practices, enforcing strong access controls and ensuring visibility through logging and monitoring.
+
+## Guidance
+
+Teams **MUST**:
+
+- follow the **[AWS][1]** or **[Azure][2]** **Well-Architected Framework** security pillar
+- encrypt all sensitive data **in transit** and **at rest** using approved cryptographic standards
+- apply the **principle of least privilege** to all **IAM policies**, roles and permissions
+- enable **centralised logging and monitoring**, integrated with the **Security Operations Centre (SOC)**
+- plan **SOC onboarding** early in the delivery lifecycle to ensure timely integration and alerting coverage
+
+## Measurement
+
+| ID    | Indicator                                     | Green                                                     | Amber                                      | Red                        |
+| ----- | --------------------------------------------- | --------------------------------------------------------- | ------------------------------------------ | -------------------------- |
+| CSA-1 | Cloud architecture reviews documented         | Reviewed and signed off                                   | Reviewed informally                        | Not reviewed               |
+| CSA-2 | Data encryption applied                       | Encrypted in transit and at rest using approved standards | Partial encryption or non-standard methods | Not encrypted              |
+| CSA-3 | IAM policies follow least privilege           | All roles scoped to minimum required access               | Some roles overly permissive               | No access control strategy |
```

Comment:
Suggest amber being some low risk roles overly permissive and red being some high risk roles.

Reply:
I think this is super important, and should stay as is.